chromium_src/chrome/android/expectations/lint-suppressions.xml

```diff
@@ -50,5 +50,12 @@
         <ignore regexp="The resource `R.color.empty_state_tab_group_border_color` appears to be unused"/>
         <ignore regexp="The resource `R.color.empty_state_icon_bg_color` appears to be unused"/>
         <ignore regexp="The resource `R.macro.default_bg_color_elev_0` appears to be unused"/>
+        <ignore regexp="The resource `R.drawable.incognito_card_bg` appears to be unused"/>
+
+        <!-- TODO(alexeybarabash): backport PasswordAccessReauthenticationHelper
```

Comment:
Fixed at  [[CR140][Android] Backport built-in Password Manager #30433](https://github.com/brave/brave-core/pull/30433)